WebContent/js/containers/JobTree.js

```diff
@@ -31,6 +31,7 @@ class JobNodeTree extends React.Component {
         super(props);
         this.state = {
             message: null,
+            filtersToggled: false,
```

Comment:
just made it **true** by default in other PR, as want focus on filters when loaded